client/components/pages/editor/sections/Hero.jsx

```diff
@@ -27,8 +27,8 @@ export default class Hero extends Component{
   render(){
     return (
       <form>
-        <ShortTextField label="Title" onChange={this.handleChange} value={this.props.data.title} name="title"/>
-        <LongTextField label="Description" onChange={this.handleChange} value={this.props.data.description} name="description"/>
+        <ShortTextField maxLength="60"label="Title" onChange={this.handleChange} value={this.props.data.title} name="title"/>
```

Comment:
I don't think this code will work. Did you test it?
(These text fields are react components so `maxLength` isn't a property. You should pass it down into the components as props and apply it to the internal `<inputs>`).